### DISCUSSION TITLE
preview raw html source code

### DISCUSSION THREAD

**Discussion by piyushkashyap2001** [remarkjs/remark#664] [Q&A] [answered]
I want to create a markdown previewer  along with an option to see the raw html source. PFA screenshot for reference.

Does react-markdown provide any option to get the raw html source from converted markdown.

<img width="1188" alt="Screenshot 2021-03-23 at 10 43 59 PM" src="https://user-images.githubusercontent.com/1442456/112189346-ecd71b00-8c29-11eb-8afb-4f533d8bc004.png">

**Comment by ChristianMurphy** [comment, marked as answer]
remark with rehype can be used to do this https://github.com/remarkjs/remark-rehype#use

> Does react-markdown provide any option to get the raw html source from converted markdown.

No, this is a non-goal for react-markdown, it focuses on react as an output. For HTML output, use the remark rehype solution above.
